WordPress/Classes/Services/PostService+Likes.swift

```diff
@@ -42,7 +42,6 @@ extension PostService {
                                                         purgeExisting: purgeExisting) {
                                         let users = self.likeUsersFor(postID: postID, siteID: siteID)
                                         success(users, totalLikes.intValue, count)
-                                        LikeUserHelper.purgeStaleLikes()
```

Comment:
Similar to https://github.com/wordpress-mobile/WordPress-iOS/pull/21028/files#r1255048104, this line is moved to the `createNewUsers` below.